src/Integrators.jl

```diff
@@ -157,11 +157,11 @@ end
 
 
 function rhs_langevin!(ΔZ::Array{CVec{N}, 4}, Z::Array{CVec{N}, 4}, ξ::Array{CVec{N}, 4},
-                        B::Array{Vec3, 4}, integrator::Langevin, sys::System{N}) where N
+                        B::Array{Vec3, 4}, circBuf, integrator::Langevin, sys::System{N}) where N
```

Comment:
For consistency, annotate type of `circBuf` here?

Also, Julia style avoids the use of camel-case `circBuf`. Another style option is snake-case, `long_variable_name`. In the case, let's use `circbuf` since it's easy to read.